app/src/main/java/com/sun/cookbook/screen/detail/DetailRecipeFragment.kt

```diff
@@ -16,7 +18,16 @@ class DetailRecipeFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_detail_recipe, container, false)
     }
 
+    override fun getRecipeDetailSuccess(recipes: RecipeDetail) {
+    }
+
+    override fun onError(exception: Exception?) {
+    }
+
     companion object {
-        fun instance() = DetailRecipeFragment()
+        private const val BUNDLE_ID = "BUNDLE_ID"
+        fun instance(id: Int) = DetailRecipeFragment().apply {
```

Comment:
id gì đặt tên rõ ràng vào e
```suggestion
        fun newInstance(id: Int) = DetailRecipeFragment().apply {
```